src/main/java/org/ambraproject/wombat/service/PeerReviewServiceImpl.java

```diff
@@ -92,26 +104,36 @@ String transformXmlToHtml(String allReviewsXml, String xsl) {
    * Aggregate the XML for all rounds of Peer Review
    *
    * @param reviewLetterItems metadata about review letters
+   * @param articleReceivedDate article received date (aka submission date)
    * @return entirety of peer review XML content
    * @throws IOException
    */
-  String getAllReviewsAsXml(List<Map<String, ?>> reviewLetterItems) throws IOException {
+  String getAllReviewsAsXml(List<Map<String, ?>> reviewLetterItems, String articleReceivedDate) throws IOException {
     List<String> reviewLetters = new ArrayList<>();
     String acceptanceLetter = "";
     for (Map<String, ?> reviewLetterItem : reviewLetterItems) {
-      String xml = getReviewXml(reviewLetterItem);
+      String xml = getContentXml(reviewLetterItem, "letter");
 
       // strip the XML declaration, which is not allowed when these are aggregated
       xml = xml.replaceAll("<\\?xml(.+?)\\?>", "");
 
-      Document parsed = Jsoup.parse(xml, "", Parser.xmlParser());
-      Elements subArticle = parsed.select("sub-article");
+      Document parsedXml = Jsoup.parse(xml, "", Parser.xmlParser());
+      parsedXml.outputSettings().escapeMode(Entities.EscapeMode.xhtml);
```

Comment:
Looks like these are chainable.  Let's set the syntax setting too.
```
      parsedXml.outputSettings().escapeMode(Entities.EscapeMode.xhtml).
          syntax(Document.OutputSettings.Syntax.xml).
          charset(CharEncoding.UTF_8);
```

Reply:
Feels odd using an html parsing library that also does xml, but the api is so simple!

Reply:
@mbowen check it, proof of concept.  I'm going to refine this and add it to the PR: https://github.com/PLOS/wombat/commit/3d6dbb0e6c412185eb7a7ebc5c731cd54861fb1e